tests/cross/src/test/scala/tests/hover/HoverScala3TypeSuite.scala

```diff
@@ -42,6 +42,35 @@ class HoverScala3TypeSuite extends BaseHoverSuite {
        |""".stripMargin.hover
   )
 
+  // We should produce a shorter type but:
+  // https://github.com/lampepfl/dotty/issues/11683
+  check(
+    "enums",
+    """|
+       |object SimpleEnum:
+       |  enum Color:
+       |   case <<Re@@d>>, Green, Blue
+       |
+       |""".stripMargin,
+    """|val Red: enums.SimpleEnum.Color.Red
```

Comment:
I didn't realize that, thanks for adding this test and creating an issue on dotty :+1: